src/stats.c

```diff
@@ -33,6 +34,42 @@ static struct {
     double user;
 } used_cpu;
 
+// log slow command
+#define BUILD_CMD_ITEM(cmd, type, access) #cmd,
+static const char * cmd_table[] = {CMD_DO(BUILD_CMD_ITEM)};
```

Comment:
`cmds` at https://github.com/eleme/corvus/blob/master/src/command.c#L64 can be reused.